### PR TITLE
Pretty print systemd services file

### PR DESCRIPTION
**Pull request by Akasurde**
Fixes: #13337

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

### CODE REVIEW

File: pkg/systemd/generate/containers.go [hunk 1]
```diff
@@ -475,7 +475,7 @@ func executeContainerTemplate(info *containerInfo, options entities.GenerateSyst
 				return "", err
 			}
 		}
-		info.ExecStart = strings.Join(startCommand, " ")
+		info.ExecStart = strings.Join(startCommand, " \\\n\t")
```
Comment (Luap99) [inline pkg/systemd/generate/containers.go:478]: I don't like this syntax, I would prefer if only options are separated with newlines and not the values, i.e.
```
--name myname \
-d \
-p 8080
```

You current code would look like:
```
--name \
myname \
-d \
-p \
8080 \
```

**Review by Luap99 — COMMENTED** [review]
You also have to update the unit tests

File: pkg/systemd/generate/containers.go (hunk 1 above)
Comment (Akasurde) [reply pkg/systemd/generate/containers.go:478]: Any clue how can I achieve this?

Comment (rhatdan) [reply pkg/systemd/generate/containers.go:478]: Only add new line before fields that start with a `-`

**Comment by rhatdan** [comment]
We will use https://github.com/containers/podman/pull/13814
Thanks @Akasurde

**Pull request closed without merging**